### ISSUE TITLE
multiple unicode 13 emojis incorrectly marked as diversity base

### ISSUE THREAD

**Issue by birarda** [joypixels/emoji-toolkit#30]
```
"1f972": {
        "name": "smiling face with tear",
        "unicode_version": 13,
        "category": "people",
        "order": 16,
        "display": 1,
        "shortname": ":smiling_face_with_tear:",
        "shortname_alternates": [],
        "ascii": [],
        "humanform": 0,
        "diversity_base": 1,
        "diversity": null,
        "diversity_children": [],
        "gender": [],
        "gender_children": [],
        "code_points": {
            "base": "1f972",
            "fully_qualified": "1f972",
            "decimal": "",
            "diversity_parent": null,
            "gender_parent": null
        },
        "keywords": [
            "uc13"
        ]
    },
```

This emoji was added to emoji.json for Unicode 13. I believe it is incorrectly marked as a diversity_base.

**Comment by birarda** [comment]
This is also true of disguised face (1f978), anatomical heart (1fac0), lungs, people hugging, and probably others I haven't hit yet.

**Comment by caseyahenson** [comment]
This is fixed in the 6.0.1 patch released just now.
